### DISCUSSION TITLE
Docker installation, but Home Assistant (and mosquitto) on a different server how to post MQTT values?

### DISCUSSION THREAD

**Discussion by Klagio** [teslamate-org/teslamate#1130] [Q&A]
Hi, I installed docker on 192.168.1.18, everything seems fine.
My Home Assistant instance is on the sale LAN but at different address 192.168.1.12, where also mosquitto server is.

How to publish on the correct mosquitto server?

**Comment by Perkinje** [comment]
Use the MQTT integration from HACS and point it at the MQTT in your docker. This is how I have it setup. Previously I used node red to listen to the MQTT in the docker and then send it to my HA MQTT  but the integration is better streamlined.
